README.md

```diff
@@ -81,17 +81,21 @@ This project implements a client-server bulletin board system using unicast sock
 ### Server
 
 1. Navigate to the server directly.
-2.  Compile the server : gcc server.c -o server
-3.  Run the server: ./server 65432
+2. Run the server : python3 server.py 65432
 
 ### Client
 
 1. Navigate to the client directory.
-2. Compile the client: gcc client.c -o client
-3. Run the client: ./client 127.0.0.1 65432
+2. Run the client : python3 client.py 127.0.0.1 65432 
```

Comment:
you dont need to provide the ip or port when running the client, that is only when connecting to server